src/Services/index.js

```diff
@@ -17,4 +23,8 @@ export {
     removeFromPlaylistService,
     getAllPlaylistsService,
     deletePlaylistService,
+    getNotesService,
+    addNoteService,
+    editNoteService,
+    deleteNoteService,
```

Comment:
Instead of importing and then exporting, you can directly use export

Example:
export { deleteNoteService } from "./NotesServices/deleteNoteService";

Also, just a suggestion:
You can create a general `noteService.js` and add all the CRUD operations for notes in the same file instead of creating a separate js file for edit, delete, and so on.